packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -8,52 +8,24 @@ const { overrideArgs } = storybookHelpers('sd-interactive');
 const { generateTemplate } = storybookTemplate('sd-interactive');
 
 /**
- * Generates basic styles for interactive elements. This should always be used in combination with `a` or `button` elements.
- * The `sd-interactive--reset` class resets the default browser styles of e. g. a button and was applied to the `button` element in the examples below.
+ * Used in interactive elements as a "quaternary" button that has no paddings and no background.
```

Comment:
* Used in interactive elements as a "quartery" button that has no paddings and no background.

Reply:
Updated